app/models/account.rb

```diff
@@ -18,7 +18,7 @@ def self.add_missing_elements(parent, xmlNode, findPath, typeName)
 
     new_accts = []
     member_map = {}
-    deletes = parent.accounts.where("accttype=?", typeName).select("id, name").collect { |a| [a.id, a.name] }
+    deletes = parent.accounts.where("accttype=?", typeName).pluck(:id, :name)
```

Comment:
@DNNX the project has not (yet) adopted the modern ruby hash syntax.  So, in this PR, let's stick to the older has syntax.